plugins/mmdblookup/mmdblookup.c

```diff
@@ -133,6 +138,48 @@ void close_mmdb(MMDB_s *mmdb) {
 	MMDB_close(mmdb);
 }
 
+const char *SEP = "!";
+
+int mmdb_get_value(
+	MMDB_entry_s *const start,
+	MMDB_entry_data_s *const entry_data,
+	char *field,
+	size_t field_len
+) {
+	// Count names in the field to construct a path, e.g.,
+	// "!continent!code" implies 2 levels of JSON object {"continent": {"code": "XX"}}.
```

Comment:
Is there a JSON object now?

Reply:
Yes, according to the example https://github.com/marselester/rsyslog/blob/master/plugins/mmdblookup/README.md#testing.